### PR TITLE
Spree::Exchange needs the model_name available as a instance method as well

### PR DESCRIPTION
**Pull request by athal7**

### CODE REVIEW

**Comment by adammathys** [comment]
:+1:

File: core/app/models/spree/exchange.rb [hunk 1]
```diff
@@ -42,5 +42,9 @@ def self.model_name
       Spree::Exchange
     end
 
+    def model_name
+      Spree::Exchange
```
Comment (magnusvk) [inline core/app/models/spree/exchange.rb]: What about calling through to `self.class.model_name` here?

**Comment by magnusvk** [comment]
:

**Comment by magnusvk** [comment]
:+1:

File: core/app/models/spree/exchange.rb (hunk 1 above)
Comment (athal7) [inline core/app/models/spree/exchange.rb]: updated

**Comment by magnusvk** [comment]
I wonder if @jhawthorn is going to ask for a spec here but personally I'm :+1: without one.

**Comment by gmacdougall** [comment]
:+1:

**Comment by athal7** [comment]
just waiting for specs to pass on the initial iteration

**Pull request merged**